app/assets/stylesheets/companies.scss

```diff
@@ -224,3 +224,7 @@ label,
   font-weight: bold;
   margin: 20px 0;
 }
+
+.url-example {
+  font-size: smaller;
```

Comment:
this is unneccessary css, preferably use .small which already exists. 
But not a blocker.